egs/cifar/v1/image/nnet3/get_egs.sh

```diff
@@ -36,6 +40,14 @@ if [ $# != 3 ]; then
   echo "                     each iteration and for combination at the end"
   echo "                     (note: there is no data held-out from training"
   echo "                     data; we use the test or dev set for that.)"
+  echo "  --crop             false. Flag to indicate that the images are not square"
```

Comment:
the default value (e.g. false) should immediately follow `--crop`. 
croped --> cropped.